base/server/python/pki/server/deployment/scriptlets/instance_layout.py

```diff
@@ -251,8 +251,11 @@ def destroy(self, deployer):
 
         # remove Tomcat instance base
         deployer.directory.delete(deployer.mdict['pki_instance_path'])
-        # remove Tomcat instance logs
-        deployer.directory.delete(deployer.mdict['pki_instance_log_path'])
+
+        # remove Tomcat instance logs only if --remove-logs or --force flags are specified
+        if deployer.mdict['pki_remove_logs'] or deployer.mdict['pki_force_destroy']:
```

Comment:
I think --force should not imply removing logs. The --force should mean removing remnant files/folders from broken installation/uninstallation that could prevent subsequent installation. The existing logs should not prevent subsequent installation (if it does, it's a bug).

So if you need to remove everything you'll need to specify both --force and --remove-logs.